fe-nc-news-react/src/App.jsx

```diff
@@ -6,20 +6,33 @@ import Home from './components/Home'
 import Topics from "./components/Topics"
 import Articles from "./components/Articles"
 import Users from "./components/Users"
+import FullArticle from './components/FullArticle'
 
 function App() {
-   const [isLoading, setIsLoading] = useState(true);
+  const [isLoading, setIsLoading] = useState(true);
+
   return (
-    <div className='App'>
+    <div className="App">
       <Header />
       <Routes>
-        <Route path='/' element={<Home />} />
+        <Route path="/" element={<Home />} />
         <Route path="/topics" element={<Topics />} />
-        <Route path="/articles" element={<Articles isLoading={isLoading} setIsLoading={setIsLoading} />} />
+        <Route
+          path="/articles"
+          element={
+            <Articles isLoading={isLoading} setIsLoading={setIsLoading} />
+          }
+        />
+        <Route
+          path="articles/:article_id"
+          element={
+            <FullArticle isLoading={isLoading} setIsLoading={setIsLoading} />
```

Comment:
We dont want to re-use the same loading state for different components - this will cause an issue when one has finished loading and the other hasn't both of the components will stop showing the 'loading' and therefore one component will have not be showing anything if the request hasnt finished by that point. Please make sure you have a separate loading state for each component as per my last feedback!